README.md

```diff
@@ -174,4 +174,9 @@ struct SubCommandTwo {
 }
 ```
 
+It is also possible to retrieve just the names of the arguments and switches 
+passed at runtime, by including the switch `--dump-args-passed` in the args, similarly
+to how `--help` can be passed. This elides any user values passed and creates an 
+output for safely collecting data for analytics.
```

Comment:
I don't think we need `--dump-args-passed` now that we have `FromArgs::to_args()`. Then it's up to the caller whether or not to call this function to trigger this functionality, with something like:

```
#[derive(FromArgs)]
struct Cmd {
    #[argh(rename = "dumped-args-passed")]
    dump_args: bool,
    ...
}

fn main() {
      let cmd = argh::from_env();
      if cmd.dump_args {
          let dumped_args = cmd.to_args();
          println!("{}", cmd.to_args().map(escape_arg).join(" "));
          std::process::exit(0);
      }
}

fn escpe_arg(&str) -> String { ... }
```